susApp/www/js/Controller/Main/Getitems.js

```diff
@@ -10,7 +10,17 @@ elephant.controller('MainpageCtrl', function($state, $ionicHistory, $scope, $htt
   /**
    * Home page view list & grid
    */
-  $scope.viewType = 'list';
```

Comment:
The changes you made are for native app. We need these changes in the web app. I would advise to take the change off this file and add it to the web app, which is /ionicApp/backend/public_html/WebApp.